sensor_msgs/Image rgb_image
---
std_msgs/Float32MultiArray image_class_probability
